src/main/java/org/jenkinsci/plugins/mesos/JenkinsScheduler.java

```diff
@@ -170,6 +170,11 @@ public synchronized boolean isRunning() {
   public synchronized void requestJenkinsSlave(Mesos.SlaveRequest request, Mesos.SlaveResult result) {
     LOGGER.info("Enqueuing jenkins slave request");
     requests.add(new Request(request, result));
+    if (driver != null) {
+      // ask mesos to send all offers, even the those we declined earlier
```

Comment:
We capitalize our sentences and end them with periods. Can you please fix here and elsewhere.

s/ask/Ask/
s/details/details./